.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
How about quarterly?

Reply:
What is `quarterly`?

Reply:
monthly seems too infrequent. weekly is a good comprise, IMO.

Reply:
It is too noisy if you get notifications from a lot of projects. And there's no easy way to turn off just that noise on the scale…